diff -up MediaSDK-intel-mediasdk-19.3.0/builder/FindOpenCL.cmake.no-icd MediaSDK-intel-mediasdk-19.3.0/builder/FindOpenCL.cmake
--- MediaSDK-intel-mediasdk-19.3.0/builder/FindOpenCL.cmake.no-icd	2019-09-30 16:51:56.000000000 +0200
+++ MediaSDK-intel-mediasdk-19.3.0/builder/FindOpenCL.cmake	2019-10-11 13:38:03.003392957 +0200
@@ -25,13 +25,11 @@ set( OCL_LIBS "" )
 
 if ( NOT OPENCL_INCLUDE MATCHES NOTFOUND )
     if ( NOT OPENCL_LIBRARY MATCHES NOTFOUND )
-        if ( EXISTS /etc/OpenCL/vendors/intel.icd )
             set ( OPENCL_FOUND TRUE )
 
             get_filename_component( OPENCL_LIBRARY_PATH ${OPENCL_LIBRARY} PATH )
 
             list( APPEND OPENCL_LIBS OpenCL )
-        endif()
     endif()
 endif()
 
